packages/connectivity/src/scp-cf/destination/service-binding-to-destination.ts

```diff
@@ -21,10 +27,48 @@ export const serviceToDestinationTransformers: Record<
   workflow: workflowBindingToDestination,
   'service-manager': serviceManagerBindingToDestination,
   xsuaa: xsuaaToDestination,
-  aicore: aicoreToDestiation
+  aicore: aicoreToDestination
 };
 
-async function aicoreToDestiation(
+/**
+ * Convenience function to create an OAuth2ClientCredentials destination from the provided service binding.
```

Comment:
[q] If one of them is basic authentication I wouldn't mention `OAuth2ClientCredentials` at all here.